python/requirements.txt

```diff
@@ -5,3 +5,4 @@ Pillow
 decorator
 astor
 opt_einsum==3.3.0
+networkx
```

Comment:
[networkx](https://networkx.org/) is pretty lightweight, and is compatible with Python on any platform, is using a  3-Clause BSD license, which is compatible with Apache2. 
It's acceptable to include it as paddlepaddle's dependency.